scenes/game/game.gd

```diff
@@ -1179,6 +1179,8 @@ func _on_advance_turn():
 
 	clear_selected_cards()
 	close_popout()
+	player_bonus_panel.visible = false
```

Comment:
This feels like something that should happen on the EventType_Strike_Cleanup event handler.

Reply:
Oh I think this was also added because of a Nine boost (see below)

Reply:
That should be fine, I just want to make sure we're not adding code around needlessly.